dags/surveymonkey_data_import_pipeline.py

```diff
@@ -0,0 +1,138 @@
+# Note: DagBag.process_file skips files without "airflow" or "DAG" in them
+
+import os
+import logging
+import json
+from datetime import timedelta
+from tempfile import TemporaryDirectory
+
+from data_pipeline.surveymonkey.surveymonkey_etl import (
+    get_survey_list
+)
+
+from data_pipeline.utils.pipeline_file_io import write_jsonl_to_file
+from data_pipeline.utils.dags.data_pipeline_dag_utils import (
+    create_dag,
+    create_python_task
+)
+
+from data_pipeline.utils.pipeline_file_io import (
+    get_yaml_file_as_dict
+)
+
+from data_pipeline.utils.data_store.bq_data_service import (
+    load_file_into_bq,
+    create_or_extend_table_schema
+)
+
+from data_pipeline.surveymonkey.get_surveymonkey_data_config import (
+    SurveyMonkeyDataConfig
+)
+
+LOGGER = logging.getLogger(__name__)
+
+SURVEYMONKEY_SECRET_FILE_ENV_VAR = "SURVEYMONKEY_SECRET_FILE"
```

Comment:
minor: below you call it `ENV_NAME` (I guess it is an env var name, so both are correct)